services/meta/client.go

```diff
@@ -900,6 +915,13 @@ func (c *Client) SetData(data *Data) error {
 	return nil
 }
 
+func (c *Client) GetData() Data {
+	c.mu.RLock()
+	defer c.mu.RUnlock()
+	d := c.cacheData.Clone()
+	return *d
+}
```

Comment:
This name should just be `Data()` instead of `GetData()`. 

https://golang.org/doc/effective_go.html#Getters